code/modules/organs/internal/carrion.dm

```diff
@@ -351,7 +351,7 @@
 	set name = "Spider call (30)"
 
 	if(last_call + 5 MINUTES > world.time)
-		to_chat(owner, SPAN_WARNING("Your maw is tired, you can only call for help every 5 minutes."))
+		to_chat(owner, SPAN_WARNING("Your maw is tired. You can only call for help every 5 minutes."))
```

Comment:
You still have the changes here that were already done in #59. These should be removed, @martinlyra should be able to help out.

Reply:
tbh it's the same code as base effectively, but yeah that's my bad and i'll make sure to un-fuck that next time